src/main/java/ladder/domain/Ladder.java

```diff
@@ -5,20 +5,20 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public class Radder {
+public class Ladder {
     public static final int MIN_HEIGHT = 1;
     private List<Line> lines;
```

Comment:
```suggestion
    private final List<Line> lines;
```

인스턴스 변수는 가능한 final로 구현한 후 값을 변경하는 상황이 발생할 때 final을 제거하는 방식으로 구현하면 어떨까?